contracts/force/force.cpp

```diff
@@ -140,6 +140,18 @@ void force::mkquali(content content, uint32_t account_id, eosio::name payer, vac
                     });
 }
 
+
+void force::editquali(uint32_t quali_id, vaccount::vaddress owner, content content, uint32_t account_id, eosio::name payer, vaccount::sig sig) {
```

Comment:
You're asking for both `account_id` and `owner`, but only one should be necessary
It would be best to follow `mkquali` and only take account_id, then use `require_vaccount` to verify the signature.